frontend/src/workbench/blocks/work-context.css

```
@@ -112,3 +112,216 @@
+  font-weight: 600;
+}
+
+.block-work-context__handoff-summary {
+  color: var(--text-muted);
+  font-size: var(--font-size-xs);
+  overflow-wrap: anywhere;
+}
+
+.block-work-context__handoff-state-pill {
+  flex: 0 0 auto;
+  border: 1px solid currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)